src/replica/snapshot_replica_test.cc

```diff
@@ -206,7 +216,118 @@ TEST_F(SnapshotReplicaTest, LeaderAndFollower) {
     sleep(2);
 }
 
-TEST_F(SnapshotReplicaTest, LeaderAndFollowerTS) {
+TEST_P(SnapshotReplicaTest, SendSnapshot) {
+    FLAGS_make_snapshot_threshold_offset = 0;
+    ::openmldb::tablet::TabletImpl* tablet = new ::openmldb::tablet::TabletImpl();
```

Comment:
@dl239 the `tablet` here seems no need to be manually deleted, as we pass it as `brpc::SERVER_OWNS_SERVICE`.

Reply:
ok